src/main/resources/db/changelog/changes/013-create-rating.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE IF NOT EXISTS rating (
+    id BIGINT AUTO_INCREMENT PRIMARY KEY,
+    item_id BIGINT NOT NULL,
```

Comment:
every NOT NULL constraint should be represented in the entity via `@Colum(nullable=false)`
it's just for the other dev to know that this value is not nullable.

Reply:
done